app/mailers/bulk_upload_mailer.rb

```diff
@@ -0,0 +1,79 @@
+class BulkUploadMailer < NotifyMailer
+  BULK_UPLOAD_COMPLETE_TEMPLATE_ID = "83279578-c890-4168-838b-33c9cf0dc9f0".freeze
+  BULK_UPLOAD_FAILED_CSV_ERRORS_TEMPLATE_ID = "e27abcd4-5295-48c2-b127-e9ee4b781b75".freeze
+  BULK_UPLOAD_FAILED_FILE_SETUP_ERROR_TEMPLATE_ID = "24c9f4c7-96ad-470a-ba31-eb51b7cbafd9".freeze
+  BULK_UPLOAD_FAILED_SERVICE_ERROR_TEMPLATE_ID = "c3f6288c-7a74-4e77-99ee-6c4a0f6e125a".freeze
+  BULK_UPLOAD_WITH_ERRORS_TEMPLATE_ID = "eb539005-6234-404e-812d-167728cf4274".freeze
+
+  def send_bulk_upload_complete_mail(user, bulk_upload)
```

Comment:
wouldn't these methods need to consume different variables for each field in the email template?

Reply:
it's possible some of them will need additional parameters. some can be pulled from `bulk_upload` eg `filename` would be `bulk_upload.filename` and `upload_timestamp` from `bulk_upload.created_at`